R/FilterStates-utils.R

```diff
@@ -1,55 +1,59 @@
 #' Initialize `FilterStates` object
 #'
-#' Initialize `FilterStates` object
-#' @param data (`data.frame`, `MultiAssayExperiment`, `SummarizedExperiment`, `matrix`)\cr
-#'   the R object which `subset` function is applied on.
-#' @param data_reactive (`function(sid)`)\cr
+#' @param data (`data.frame` or `MultiAssayExperiment` or `SummarizedExperiment` or `matrix`)
+#'   the `R` object which `subset` function is applied on.
+#' @param data_reactive (`function(sid)`)
 #'   should return an object of the same type as `data` or `NULL`.
 #'   This object is needed for the `FilterState` shiny module to update
 #'   counts if filtered data changes.
 #'   If function returns `NULL` then filtered counts
 #'   are not shown. Function has to have `sid` argument being a character which
 #'   is related to `sid` argument in the `get_call` method.
-#' @param dataname (`character(1)`)\cr
+#' @param dataname (`character(1)`)
 #'   name of the data used in the expression
 #'   specified to the function argument attached to this `FilterStates`.
-#' @param datalabel (`character(0)` or `character(1)`)\cr
+#' @param datalabel (`NULL` or `character(1)`)
 #'   text label value.
 #' @param ... (optional)
 #'   additional arguments for specific classes: keys.
 #' @keywords internal
-#' @export
 #' @examples
-#' library(shiny)
```

Comment:
@kartikeyakirar the same topic as discussed with @pawelru due to the transition of shiny from Depends to Imports
https://github.com/insightsengineering/teal.slice/issues/529
We then need to keep `library(shiny)`

Reply:
at the current state the removal is correct - this line is redundant
We might reintroduce it in the aforementioned task. Before or after release.

Reply:
ok